.github/workflows/check_smoke_tests.yml

```diff
@@ -64,12 +64,15 @@ jobs:
           docker run -d --name rl-service \
             --network="host" \
             -e DB_URI=$DB_URI \
+            -e API_ROOT_PATH="/api" \
             rl-service-image
 
+          BASE_URL="http://localhost:8080/api"
+
```

Comment:
More a curiosity than anything, but in this case do the BASE_URL and API_ROOT need to have the same extension? I.e. if the BASE_URL were `localhost:8080` but then the API_ROOT were `/api` would that cause problems? Wondering mostly because as written this seems to move exposure to the whole app behind the API path, which may or may not be the intended functionality. My intuition is that anything prefixed with `/api` is something that an API-specific consumer would access--such as some pipeline system reaching out for a request--whereas the root of `localhost:8080/` (no `/api` included) would be used by e.g. us or a user messing around on their local with the app open. Is that right?

Reply:
There is a dependency between the two, and maybe its better to create two variables to show that.
```shell
API_ROOT="/api"
BASE_URL="http://localhost:8080${API_ROOT}"
```

The intent is that all API calls and at least access to the openapi schema (`/api/openapi.json`), will be behind the prefix.  Micheal didn't explicitly state whether the docs (swagger and redoc) should be behind this root path or not, but to me, it made sense to do that as well (this is certainly open for discussion if you disagree).

In `main`, the root URL isn't be used for anything and is currently returning a 404.  I'm wondering if we should add a redirect to the docs?  Thoughts?

Reply:
Yeah no strong feelings on whether the docs should be behind the root, I could see good justification either way. good find on the root URL returning a 404, though--I think redirecting to the docs removes what might seem like a "failure point" to a potential user and keeps them firmly directed towards our instruction and education.